### ISSUE TITLE
Your code does not work for new version of keras

### ISSUE THREAD

**Issue by Ameybhile** [Shobhit20/Image-Captioning#4]
def create_model(self, ret_model = False):
	       
		image_model = Sequential()
		image_model.add(Dense(EMBEDDING_DIM, input_dim = 4096))
		image_model.add(Activation('relu'))
		image_model.add(RepeatVector(self.max_length))
		

		lang_model = Sequential()
		lang_model.add(Embedding(self.vocab_size, 256, input_length=self.max_length))
		lang_model.add(LSTM(256,return_sequences=True))
		lang_model.add(TimeDistributed(Dense(EMBEDDING_DIM)))
		

		model = Sequential()
		keras.layers.concatenate([image_model, lang_mode])(axis=-1)
		model.add(LSTM(1000,return_sequences=False))
		model.add(Dense(self.vocab_size))
		model.add(Activation('softmax'))
		

		print ("Model created!")

		if(ret_model==True):
		    return model

		model.compile(loss='categorical_crossentropy', optimizer='rmsprop', metrics=['accuracy'])
		return model

	def get_word(self,index):
		return self.index_word[index]

Can you mdify this part
